frontend/src/App.js

```diff
@@ -20,6 +21,7 @@ function App() {
 						</Route>
 						<Route path="login" element={<LoginPage />} />
 						<Route path="sign-up" element={<SignUpPage />} />
+						<Route path="create-bot" element={<CreateBot />} />
```

Comment:
pagina de create bot trebuie sa fie in interiorul rutei cu RequireAuth, la fel ca main

altfel poti sa creezi bot fara sa fii autentificat